doc/cephadm/install.rst

```diff
@@ -45,7 +46,7 @@ curl-based installation
   
   .. prompt:: bash #
 
-   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/octopus/src/cephadm/cephadm
+   curl --silent --remote-name --location https://github.com/ceph/ceph/raw/master/src/cephadm/cephadm
```

Comment:
don't know. This is the user facing documentation and I'd like to prevent anyone from downloading the latest master for their production systems. Maybe change this to pacific, as soon as the GA release is out?

Reply:
The problem is that documentation tends to get cached all over the place, so it's likely that even when we change it to "pacific" it'll still be stuck at "octopus" in all of the cached copies across the web for years to come.

Is there a way to ensure that this URL always points at the latest released version branch?

Reply:
Or maybe...just add a "cephadm" branch to the repo that points to the branch that we do want people to download this from? Then we can just update that branch whenever we want to release a new version?